### PR TITLE
Update bcftools-snvphyl-plugin to 1.6

### PR DESCRIPTION
**Pull request by nsoranzo**
Depend on bcftools instead of conflicting with it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by Takadonet** [comment]
Thanks for this @nsoranzo  I will take a look sometime tomorrow. Cheers!

**Comment by Takadonet** [comment]
lol. That works too

File: recipes/bcftools-snvphyl-plugin/meta.yaml [hunk 1]
```diff
@@ -29,13 +30,8 @@ requirements:
...
   url: https://github.com/samtools/bcftools/releases/download/{{ version }}/bcftools-{{ version }}.tar.bz2
-  md5: e43efe3df5f9c55acb4b48cee214d281
-  patches:
-    - b37b03e815cd4f25af70a57c08449f8c719dcc6a.patch # from https://github.com/samtools/bcftools/pull/645
-
+  sha256: 293010736b076cf684d2873928924fcc3d2c231a091084c2ac23a8045c7df982
 
 test:
   commands:
-    - bcftools -h
```
Comment (bgruening) [inline recipes/bcftools-snvphyl-plugin/meta.yaml]: @nsoranzo is there any reason you removed the tests?

Comment (nsoranzo) [reply recipes/bcftools-snvphyl-plugin/meta.yaml]: Yes, these are already tested in the main bcftools package, this now provides only the plugin.

**Pull request merged**